templates/user/edit-profile.html

```diff
@@ -177,6 +199,65 @@
             });
         });
     </script>
+    <script src="{{ static('libs/clipboard/clipboard.js') }}"></script>
+    <script src="{{ static('libs/clipboard/tooltip.js') }}"></script>
+    <script type="text/javascript">
+        $(function () {
+            $('#generate-scratch-codes-button').click(function (event) {
+                event.preventDefault();
+                if (confirm("{{ _('Are you sure you want to generate or regenerate a new set of scratch codes? ') }}" +
+                            "{{ _('This will invalidate any previous scratch codes you have. ') }}" +
+                            "{{ _('You will not be able to view your scratch codes after you leave this page!') }}")) {
```

Comment:
Changed back to using string concatenation instead of escaping newlines since there were large spaces between each sentence.